draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1136,7 +1136,7 @@ Reliability:
 Source:
 : UUID version 1 and 6 both utilize a Gregorian epoch timestamp while UUIDv7
   utilizes a Unix Epoch timestamp. If other timestamp sources or a custom timestamp
-  epoch are required UUIDv8 SHOULD be used.
+  epoch are required UUIDv8 MUST be used.
```

Comment:
Since all of the time UUIDs only use either Gregorian or Unix timestamp, and v8 is the only option where you can do something else, it should be fine to convert this to a MUST.